packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.alb-fargate-service-https.ts

```diff
@@ -27,6 +27,7 @@ new ApplicationLoadBalancedFargateService(stack, 'myService', {
     zoneName: 'example.com.',
   }),
   redirectHTTP: true,
+  circuitBreaker: { rollback: true, useExplicitEcsDeploymentController: false },
```

Comment:
I would prefer to have a new test written instead of just updating the current test.

Reply:
Done